content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -0,0 +1,278 @@
+---
+title: Unified Service Tagging
+kind: documentation
+aliases:
+- /tagging/unified_service_tagging
+further_reading:
+- link: "/tagging/using_tags"
+  tag: "Documentation"
+  text: "Learn how to use tags in the Datadog UI"
+- link: "/blog/autodiscovery-docker-monitoring"
+  tag: "Blog"
+  text: "Learn more about Autodiscovery"
+---
+
+## Overview
```

Comment:
This is a very long overview with section that are specific to products. We might want to revisit the structure so that the overview remain generic and then have more targeted sections.

WDYT?